Classes/Domain/Repository/PageRepository.php

```diff
@@ -3,8 +3,33 @@
 
 namespace WebVision\WvDeepltranslate\Domain\Repository;
 
-use TYPO3\CMS\Extbase\Persistence\Repository;
+use TYPO3\CMS\Core\Database\Connection;
+use TYPO3\CMS\Core\Database\ConnectionPool;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
 
-final class PageRepository extends Repository
+final class PageRepository
 {
+    /**
+     * @param array{uid: int} $targetLanguage
+     */
+    public function markPageAsTranslatedWithDeepl(int $pageId, array $targetLanguage): void
+    {
+        GeneralUtility::makeInstance(ConnectionPool::class)
+        ->getConnectionForTable('pages')
+        ->update(
+            'pages',
+            [
+                'tx_wvdeepltranslate_content_checked' => 1,
```

Comment:
Why is here content_checked set to 1?
After automatic translation content_checked should be 0 (==false)
```suggestion
                'tx_wvdeepltranslate_content_checked' => 0,
```